nss-day-cohort-29/nutshell-midnight-dream
# Welcome and Register Pages

# Description

- Welcome page basic HTML Elements inside the CORRECT tags.
- If not a member, user can click on the word "here" and will be redirected to a "register" page
- BASIC styling specific ONLY to the containers I have made (Shouldn't affect anyone else's HTML elements

## Type of change

Please delete options that are not relevant.



# Testing Instructions for Change Made

1. ```git fetch all```
2. ```git checkout user_welcome_and_register
3. Check to see if welcome page HTML elements are spelled correctly and displayed in correct order
4. *** Check if every element is appended CORRECTLY and won't mess others up down the line***
5. Click the "here" text and make sure it takes the user to a registration page requesting a username and user email
6. *** Check if every element is appended CORRECTLY and won't mess others up down the line for the registration page too***

# Checklist:
